bundles/org.openhab.binding.mikrotik/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="mikrotik:routeros">
+		<instruction-set targetVersion="1">
```

Comment:
The version also need to be added to the `interface` Thing as property.

```
<properties>
    <property name="thingTypeVersion">1</property>
  </properties>
```

Reply:
Only this comment is left ^^, not sure if you missed it

Reply:
<thing-type id="interface">

Reply:
Gentle ping @asp24 we are not far from the next milestone. Would you be able to look at this?